zebra-chain/src/sprout/joinsplit.rs

```diff
@@ -14,6 +15,35 @@ use crate::{
 
 use super::{commitment, note, tree};
 
+/// Compute the [h_{Sig} hash function][1] which is used in JoinSplit descriptions.
+///
+/// `random_seed`: the random seed from the JoinSplit description.
+/// `nf1`: the first nullifier from the JoinSplit description.
+/// `nf2`: the second nullifier from the JoinSplit description.
+/// `joinsplit_pub_key`: the JoinSplit public validation key from the transaction.
+///
+/// [1]: https://zips.z.cash/protocol/protocol.pdf#hsigcrh
+pub(super) fn h_sig(
+    random_seed: &[u8],
+    nf1: &[u8],
+    nf2: &[u8],
+    joinsplit_pub_key: &[u8],
```

Comment:
Same here?
https://docs.rs/x25519-dalek/1.2.0/x25519_dalek/struct.PublicKey.html

Reply:
Done in 54aa38c